semanticarts/.htaccess

```diff
@@ -0,0 +1,18 @@
+RewriteEngine on
+
+# w3id.org/semanticarts/gist
+# This is the gist namespace, we are redirecting it to a web page
+# RewriteRule ^semanticarts/gist$ https://ontologies.semanticarts.com/o/namespace.html [R=302,L]
+RewriteRule ^semanticarts/gist/$ https://ontologies.semanticarts.com/o/namespace.html [R=302,L]
```

Comment:
you need to remove `semanticsarts/` from the rewrite rule, as you already have the folder with `semanticsarts`. Otherwise you will be creating the w3id for https://w3id.org/semanticsarts/semanticsarts/gist